dist/index.js

```diff
@@ -133,16 +133,30 @@ async function build(tag, file, additionalDockerArguments, contextPath) {
   }
 }
 
+/**
+ * Retrieve the digest of an image. Assumes it has already been pushed to the registry.
+ * @param {string} imageId - The id of the image.
+ * @return {string} - The digest of the image, falsy otherwise.
+ */
+async function getDigest(imageId) {
+  try {
+    return cp.execSync(`docker image inspect "${imageId}" -f '{{index .RepoDigests 0}}' | cut -d'@' -f2`)
```

Comment:
I had to change this from the previous usage of image tag as there is a [bug](https://github.com/docker/cli/issues/728) that forbid to retrieve digest of image when using a tag.